src/UserBundle/Controller/UserController.php

```diff
@@ -398,6 +398,40 @@ public function updateAction(Request $request, User $user)
         return new Response($json);
     }
 
+
+        /**
+     * Edit a user {id} with data in the body
+     *
+     * @Rest\Post("/users/{id}/updateLanguage", name="update_user")
+     * @Security("is_granted('ROLE_USER_MANAGEMENT_READ')")
```

Comment:
No need for this security, the only person who can change one's language is the user itself. That's what you should check (in the controller)